packages/matchbox/src/components/ListBox/useOptionConstructor.js

```diff
@@ -67,15 +67,15 @@ function useOptionConstructor({ options, value, onSelect, open, placeholder }) {
   }, [focused]);
 
   React.useLayoutEffect(() => {
-    if (keysSoFar) {
+    if (keysSoFar && options) {
       const index = options.findIndex(option =>
-        option.props.value.toLowerCase().startsWith(keysSoFar),
+        option.props.children.toLowerCase().startsWith(keysSoFar),
```

Comment:
Just to clarify, this is searching children (the option label) instead of the value, right?

Reply:
Yeah